ui/admin/src/app/libraries/library.ts

```diff
@@ -32,4 +48,36 @@ export class Library {
     }
     return openinghours;
   }
+
+  update(obj) {
+    Object.assign(this, obj);
+  }
+
+  deleteException(index) {
+    this.exception_dates.splice(index, 1);
+  }
+
+  sortExceptions() {
+    this.exception_dates.sort(function(a, b) {
+        const keyA = new Date(a.start_date);
```

Comment:
moment can be used. Something like: `return moment(a.start_date).substract(moment(b.start_date))`